docs/site-generator/Makefile

```diff
@@ -0,0 +1,155 @@
+# Default shell
+SHELL := /bin/bash
+
+# Config
+SHARED := $(shared)
+export BASEURL := $(baseurl)
+export OUTPUT := $(output)
+export OWNER := $(owner)
+export REPOSITORY := $(repository)
+export VERSION := $(version)
+export LANGUAGES := $(languages)
+
+URL_TARBALL := https://github.com/$(OWNER)/$(REPOSITORY)/archive/$(VERSION).tar.gz
+REPO_TMP_PATH := $(SHARED)/$(OWNER)-$(REPOSITORY)-$(VERSION)
```

Comment:
you should not be using shared for anything that is not putting shared assets. This are tmp stuff. For that you already have a folder in tmp where the project is and a final destination provided by docsrv.